### PR TITLE
Book collection Page

### PR DESCRIPTION
**Pull request by techEmprez**
### **The following were done in this project:**

- Created a plain HTML page with no styling
- Created an array of objects for the book lists
- Created a function that adds and removes new books with title and authors
- When a user clicks add book, there is a display of the new book

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```
Comment (TedLivist) [inline README.md]: - [x] Let's get rid of dummy text from the template and include more specific instructions on how to set up and run the app locally.

File: README.md [hunk 2]
```diff
@@ -30,27 +26,27 @@ To get a local copy up and running follow these simple example steps.
 
 ### Install
 
+- To access this site, you can clone it to you local environment
+
 ### Usage
 
 ### Run tests
 
 ### Deployment
```
Comment (TedLivist) [inline README.md]: - [x] I would recommend you clean up your documentation. If these sections are not in use, then kindly remove them.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @techEmprez and @mahdimahdawi,

Good job so far!
✔️ Books are stored and retrieved from localStorage
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (techEmprez) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (techEmprez) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -1,73 +1,40 @@
...
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
 
 This project is [MIT](./MIT.md) licensed.
```
Comment (Bondok6) [inline README.md:40]: - [OPTIONAL] I noticed you used the `MIT.md` template great job 👍, but you forgot to customize it, Kindly add your name and customize it 👌💯

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

image

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @techEmprez  and @mahdimahdawi 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file ✅💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**